src/steamship/agents/memory/chathistory.py

```diff
@@ -160,3 +163,30 @@ def messages(self) -> List[Block]:
     @property
     def client(self) -> Client:
         return self.file.client
+
+    def messages_as_string(
+        self,
+        memory_window_strategy: MemoryStrategy,
```

Comment:
Suggestion to name this 

MemoryFetcher # Used to fetch relevant messages from the chat history